lib/matplotlib/axes/_base.py

```diff
@@ -1566,6 +1566,11 @@ def set_prop_cycle(self, *args, **kwargs):
         self._get_patches_for_fill.set_prop_cycle(prop_cycle)
 
     def get_aspect(self):
+        """
+        Return the aspect of the axis scaling.
+
+        This is either "auto" or a float giving the ratio of y-unit to x-unit.
```

Comment:
I guess we use `x/y-unit` below, but that is confusing as this has nothing to do with units.  
```suggestion
        This is either "auto" or a float giving the ratio of y-data to x-data.
```

Reply:
I don't think "data" is quite right either. It's the ratio of the screen size of 1 unit on the y-axis to 1 unit on the x-axis. But how to write this concise?

Reply:
y/x-coordinate?

Reply:
I've chosen y/x-scale. Coordinate sounds too much like absolute numbers (e.g. ~ if xmax=1, ymax=1000, then y/x-coordinate would be 1000/1.)